.travis.yml

```diff
@@ -1,13 +1,25 @@
 language: python
 python:
   - '2.7'
+  - '3.6'
 sudo: required
 install:
```

Comment:
@afaulconbridge you may want to add a install step to install pytest and codecov package just to not put them in the requirements file otherwise the test will fail

Reply:
Possibly. Or maybe they should be as an optional dependency? e.g. see https://stackoverflow.com/a/43090648